src/Tools/MicroBuild/publish-assets.ps1

```diff
@@ -37,13 +37,11 @@ try
     
     switch ($branchName)
     {
-        "dev15.0.x" { } 
-        "dev15.1.x" { } 
-        "dev15.2.x" { } 
-        "dev15.3-preview1" { } 
-        "dev15.3-preview2" { }
-        "master" { } 
-        "post-dev15" { } 
+        "dev15.0.x" { }
+        "dev15.1.x" { }
+        "dev15.2.x" { }
```

Comment:
Yes, you can clean up the old ones. What matters here is the content of the file in the branch itself.